crmsh/ui_cluster.py

```diff
@@ -301,9 +301,8 @@ def looks_like_hostnames(lst):
                             help='Answer "yes" to all prompts (use with caution, this is destructive, especially those storage related configurations and stages. The /root/.ssh/id_rsa key will be overwritten unless the option "--no-overwrite-sshkey" is used)')
         parser.add_argument("-n", "--name", metavar="NAME", dest="cluster_name", default="hacluster",
                             help='Set the name of the configured cluster.')
-        parser.add_argument("-N", "--nodes", metavar="NODES", dest="nodes",
+        parser.add_argument("-N", "--nodes", metavar="NODES", dest="nodes", action="append", default=[],
```

Comment:
Cosmetic improvement for the grammar in UI

```suggestion
        parser.add_argument("-N", "--node", metavar="NODENAME", dest="node", action="append", default=[],
```

Reply:
Changed, except dest changed as node_list